guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -184,4 +178,13 @@ endif::[]
 # {foreman-installer}
 ----
 endif::[]
+ifdef::foreman-el,katello,satellite[]
+. On {EL} 8: Switch to the PostgreSQL 13 module:
```

Comment:
For Katello/Foreman, this *has* to happen before "Ensure the module streams are enabled" -- otherwise the rest of the upgrade will fail.